specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimtags.json

```diff
@@ -1246,7 +1246,282 @@
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Gets the details of the tag specified by its identifier.",
+            "schema": {
+              "$ref": "./apimtags.json#/definitions/TagContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tag_AssignToProduct",
+        "description": "Assign tag to the Product.",
+        "x-ms-examples": {
+          "ApiManagementCreateProductTag": {
+            "$ref": "./examples/ApiManagementCreateProductTag.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimproducts.json#/parameters/ProductIdParameter"
+          },
+          {
+            "$ref": "#/parameters/TagIdParameter"
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": false,
+            "description": "The entity state (Etag) version of the Tag to update. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+            "type": "string"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Tag was assigned to the Product.",
+            "schema": {
+              "$ref": "./apimtags.json#/definitions/TagContract"
+            }
+          },
+          "200": {
+            "description": "Tag is already assigned to the Product.",
+            "schema": {
+              "$ref": "./apimtags.json#/definitions/TagContract"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tag_DetachFromProduct",
```

Comment:
`DELETE` operations must be named as `Tag_Delete`

Reply:
The PR was signed off by ARM https://github.com/Azure/azure-rest-api-specs/pull/2257. We are just moving operation into different file, as it caused the Documentation CI to break